inlang/source-code/manage/src/components/InlangUninstall.ts

```diff
@@ -3,13 +3,19 @@ import { html } from "lit"
 import { openRepository, createNodeishMemoryFs } from "@lix-js/client"
 import { customElement, property } from "lit/decorators.js"
 import { TwLitElement } from "../common/TwLitElement.js"
-import { browserAuth, getUser } from "@lix-js/server"
```

Comment:
dont you have to rewrite the getUser call in this file too

Reply:
line 173

Reply:
Good catch - how can this pass the ci - gonna fix it

Reply:
Just talked with @NiklasBuchfink we fix this in - we will fix the ci first https://linear.app/opral/issue/INWEB-55/update-or-deprecate-manageinlangcom and i will update the pr with the changes.